mesonbuild/interpreter/interpreter.py

```diff
@@ -1095,6 +1095,9 @@ def func_get_option(self, nodes: mparser.BaseNode, args: T.Tuple[str],
             raise InterpreterException('Having a colon in option name is forbidden, '
                                        'projects are not allowed to directly access '
                                        'options of other subprojects.')
+        if optname.startswith('.') or sum(1 for c in optname if c == '.') > 1:
```

Comment:
```suggestion
        if optname.startswith('.') or optname.count('.') > 1:
```

Reply:
And I'm considered here as a Python expert... 🤦